README.adoc

```diff
@@ -67,14 +68,16 @@ Other options include Homebrew and Zip bundles. Please checkout the full list of
 
 == Compatibility
 
-Neo4j-Migrations can be used against Neo4j 3.5, all Neo4j 4.x versions upto 4.4 and Neo4j 5, including all current Neo4j-Aura versions.
+Neo4j-Migrations can be used against Neo4j 3.5, all Neo4j 4 versions from 4.1 up to 4.4 and Neo4j 5, including all current Neo4j-Aura versions.
+
+NOTE: Neo4j 4.0 is only partially supported since version 2.0 due to the fact that Neo4j Java Driver does not support it fully anymore since 5.x. If you need to use this library with Neo4j 4.0 (which you shouldn't, because 4.0 is out of support anyway), look at the latest 1.x version of Neo4j-Migrations.
 
 // tag::compatibility[]
 The Core API and the JVM based version of the CLI module of Neo4j-Migrations requires at least Java 17 or higher since version 2.0.
 Neo4j-Migrations can safely be used on both the class- and module-path.
 Native binaries are provided for 64bit versions of macOS, Linux and Windows. The native binaries don't require a JVM to be installed.
 
-For a version compatible with JDK 8, check the 1.x releases. We still do maintain the latest minor, including support for older versions of Spring Boot (prio to Spring Boot 3)
+For a version compatible with JDK 8, check the 1.x releases. We still do maintain the latest minor, including support for older versions of Spring Boot (prio to Spring Boot 3). This is also the versions you should be using against Neo4j 4.0.
```

Comment:
```suggestion
For a version compatible with JDK 8, check the 1.x releases. We still do maintain the latest minor, including support for older versions of Spring Boot (prio to Spring Boot 3). These are also the versions you should be using against Neo4j 4.0.
```

Optional, but I think it makes it read a bit better.

Reply:
Indeed it does. Thank you!